cdc/sinkv2/eventsink/txn/worker.go

```diff
@@ -67,49 +82,91 @@ func (w *worker) runBackgroundLoop() {
 		defer w.wg.Done()
 		defer func() {
 			if err := w.backend.Close(); err != nil {
-				log.Info("transaction sink backend close fail",
+				log.Info("Transaction sink backend close fail",
+					zap.String("changefeedID", w.changefeed),
+					zap.Int("workerID", w.ID),
 					zap.Error(err))
 			}
 		}()
-		w.timer = time.NewTimer(w.backend.MaxFlushInterval())
+		defer func() {
+			var r interface{}
+			if r = recover(); r == nil {
```

Comment:
```suggestion
			if r = recover(); r == nil {
```
Should we print some logs here?

Reply:
There is already a log for existing from the for loop.